src/script/index.js

```diff
@@ -42,11 +42,38 @@ function getMolfile() {
 		{ ignoreErrors: true });
 }
 
+
+function getRepresentationInFormat(format) {
```

Comment:
nit: if it's not too much, let's try and add jsdoc strings (or similar) to newer code

Reply:
Sure, done